docs/src/main/sphinx/installation/cli.rst

```diff
@@ -396,6 +396,15 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
+Auto suggestion
+^^^^^^^^^^^^^^^
+
+The CLI generates autocomplete suggestions based on command history.
+
+Press :kbd:`→` to accept the suggestion and replace the current command line
+buffer. Press :kbd:`Option+→` to accept only the next keyword. Continue typing
```

Comment:
Option is only on a Mac keyboard ... what about windows and linux machines? Is it ALT ?  Or Windows key?

Reply:
@nineinchnick do you know the answer to this?

Reply:
It will be `Ctrl+→` in Windows.